src/client/scripts/cardModal.js

```diff
@@ -0,0 +1,72 @@
+"use strict";
+/* global Cards:false SnackbarMethods:false UserMethods:false */
+
+Template.cardModal.helpers({
+    cardModalInfo: function(_id) {
+        return Cards.findOne({"_id": _id});
+    }
+});
+
+Template.cardModal.events({
+    "click #submitCommentButton": function(eve){
+        eve.preventDefault();
+        var comment = eve.target.form[0].value;
+        var image = null;
+
+        if(!comment || comment.length <= 4)
+            SnackbarMethods.DisplayMessage("Enter a more valuable comment",3000);//eslint-disable-line
```

Comment:
Just newline, don't disable line...

As a side note, I'm kinda feeling we should bump up max line length to 120 instead of 80